salt/metalk8s/containerd/configured.sls

```diff
@@ -1,3 +1,8 @@
+{# One day this should be transferred from the fileserver #}
+{% set pause_image_archive =
+    '/srv/scality/metalk8s-dev/salt/metalk8s/containerd/files/pause-3.1.tar'
```

Comment:
Hard-coded path for now. Not great, but don't know how to do better for now.

Reply:
Why not `file.copy` it somewhere else and then execute.
Because that also mean you can only run this state on the salt master or at least somewhere where you extracted the iso

Reply:
Ugh, right, I can do this one :blush:

Reply:
So, yeah, I guess, no, I can't. Reason: at the time of execution, there is no `salt-master`. There is no fileserver. Etc.

Reply:
So either we need to know whether we're in 'local' mode or in 'normal' mode, and dispatch accordingly, but even then, the source location in 'local' mode needs to be found somehow :(

Reply:
My bad that doesn't work with `file.copy` this state is only to copy file localy on the machine but with file.managed you copy it even in masterless mode.

Reply:
I'll give that a try...

Reply:
@TeddyAndrieux I'd like to get this one in and handle the follow-up through #639

Reply:
Ok np